client/my-sites/theme/hooks/use-bundle-settings.tsx

```diff
@@ -24,11 +24,11 @@ const WooOnPlansIcon = () => (
 );
 
 /**
- * Hook to get the bundle settings for a given theme.
- * If the theme doesn't have a sotfware set defined, it returns `null`.
+ * Hook to get the bundle settings for a software set.
  */
-const useBundleSettings = ( themeId: string ): BundleSettings | null => {
-	const themeSoftwareSet = useSelector( ( state ) => getThemeSoftwareSet( state, themeId ) );
+export const useBundleSettingsFromThemeSoftwareSet = (
```

Comment:
Hi @okmttdhr! Thank you for adding me as a reviewer here.
We did a similar thing here in parallel. https://github.com/Automattic/wp-calypso/pull/84886/files#r1422629619

Basically, in my case, I used the `useBundleSettings` getting by the `themeSoftware`, and I created another one with the name `useBundleSettingsByTheme` to get by theme.

I also changed the default export to a named export because I noticed later that it's the way it's used in the majority of the hooks in Calypso.

WDYT? I think it would be good if we apply the same code in both PRs to avoid issues.

Reply:
Thank you for letting me know, @renatho! 😄
I will modify my approach to align with your https://github.com/Automattic/wp-calypso/pull/84886, as I prefer `useBundleSettings` if we are retrieving `bundleSettings` from `themeSoftware` on a one-to-one basis.

Reply:
📝 I tried to be minimalist in applying #84886: https://github.com/Automattic/wp-calypso/pull/85013/commits/d87ac5485d810bad40c7602a25560ad6a9bdfb07